src/pages/Sidebar/HappinessForm.tsx

```diff
@@ -0,0 +1,213 @@
+        end: formatDate(
+          new Date(
+            new Date().getFullYear(),
+            new Date().getMonth(),
+            new Date().getDate(),
+          ),
+        ),
+      })
+      .then((res) => res.data);
+  });
+
+  // react to initial query
+  useEffect(() => {
+    // console.log("is updating");
+    if (isLoading || data === undefined) {
+      // TODO from design: loading state for this submission box
+      // console.log(`Loading!`);
+      setSubmissionStatus(UPDATING);
+    } else if (isError) {
+      // TODO from design: error state for this submission box
+      // console.log("Error!");
+      setSubmissionStatus(ERROR);
+    } else {
+      const idx: number = radioValue === 1 ? 0 : 1;
+      if (data[idx] === undefined) {
+        setSubmissionStatus(UNSUBMITTED);
+        setHappiness(-1);
+        setComment("");
+      } else {
+        setSubmissionStatus(UPDATED);
+        setHappiness(data[idx].value);
+        setComment(data[idx].comment);
+      }
+    }
+  }, [isLoading, selDate]);
+
+  return (
+    <>
+      <div className="mb-4 flex w-full justify-center">
+        <button
+          className={
+            "w-1/2 rounded-l-lg border border-light_gray p-1 text-sm font-medium " +
+            (radioValue === 1
+              ? "bg-yellow text-secondary"
+              : "bg-white text-dark_gray")
+          }
+          onClick={() => {
+            setRadioValue(1);
+          }}
+        >
+          <label>Yesterday</label>
+        </button>
+        <button
+          className={
+            "border-right w-1/2 rounded-r-lg border border-l-0 border-light_gray p-1 text-sm font-medium " +
+            (radioValue === 2
+              ? "bg-yellow text-secondary"
+              : "bg-white text-dark_gray")
+          }
+          onClick={() => {
+            setRadioValue(2);
+          }}
+        >
+          <label>Today</label>
+        </button>
+      </div>
+      <div className="mb-4 rounded-xl bg-white p-4">
+        <div className="text-sm font-medium text-dark_gray">
+          {selDate.toLocaleString("en-us", { weekday: "long" })}
+        </div>
+        <div className="flex w-full items-center">
+          <div className="text-raisin-600 w-4/5">
+            <div className="text-lg font-semibold">
+              {selDate.toLocaleDateString("en-us", { month: "long" }) +
+                " " +
+                selDate.getDate()}
+            </div>
+          </div>
+          <HappinessNumber
+            value={happiness}
+            onChangeValue={(n: number) => {
+              const val: string = n.toString();
+              if (parseFloat(val) >= 0) {
+                setHappiness(parseFloat(val));
+              }
+            }}
+            editable={true}
+            sidebar={true}
+          />
+        </div>
+        <div className="mt-1.5 flex w-full justify-center">
+          {/* 662 = height in px of other sidebar elements */}
+          <TextareaAutosize
```

Comment:
The border color around this text area and the text color do not match the design.